tests/testthat/test-trace-collapse-children.txt

```diff
@@ -0,0 +1,17 @@
+Full:
+█
+└─f()
+  └─eval(quote(g()), env())
+    └─eval(quote(g()), env())
+      └─g()
+
+Collapsed:
+█
+└─f()
+  └─[ eval(...) ] ... +1
+    └─g()
+
+Trail:
+ ─f()
+ ─[ eval(...) ] ... +1
```

Comment:
`... +1` is very concise. I think I bought this up before, but do you think we could use some words here? Like `... and 1 more call`.

Reply:
tibble does "... with 53,930 more rows" so maybe borrow that language for familiarity?

Reply:
ok let's try it out. Any objections to make this configurable?

Reply:
The downside of making anything configurable is that we now have to test under both configurations. So the added complexity has to have some clear payoff.